ui-lib/src/androidTest/java/co/electriccoin/zcash/ui/common/FlowExtTest.kt

```diff
@@ -54,13 +54,13 @@ class FlowExtTest {
     @FlakyTest
     @Test
     fun stressTest() = runBlocking {
-        for (i in 0..10) {
+        repeat(10) {
```

Comment:
Note the `repeat` is zero-based and exclusive at the end, thus this is changing the loop iterations from 11 to 10, but it's alright for its test intention here and more clarity.